app/src/main/java/org/wikipedia/talk/template/AddTemplateActivity.kt

```diff
@@ -245,16 +240,6 @@ class AddTemplateActivity : BaseActivity(), LinkPreviewDialog.Callback, UserMent
         FeedbackUtil.showError(this, t)
     }
 
-    override fun onLinkPreviewLoadPage(title: PageTitle, entry: HistoryEntry, inNewTab: Boolean) {
-        startActivity(if (inNewTab) PageActivity.newIntentForNewTab(this, entry, title) else
-            PageActivity.newIntentForCurrentTab(this, entry, title, false))
-    }
-
-    override fun onLinkPreviewAddToList(title: PageTitle) {
-        ExclusiveBottomSheetPresenter.show(supportFragmentManager,
-                AddToReadingListDialog.newInstance(title, Constants.InvokeSource.TALK_REPLY_ACTIVITY))
```

Comment:
Same here.

Reply:
Using `TALK_REPLY_ACTIVITY` was already incorrect here. So, even if using the default `LINK_PREVIEW_MENU` is also incorrect, at least it will be incorrect with less code.